fr32/src/padding.rs

```diff
@@ -563,6 +563,7 @@ need to handle the potential bit-level misalignments:
 // offset and num_bytes are based on the unpadded data, so
 // if [0, 1, ..., 255] was the original unpadded data, offset 3 and len 4 would return
 // [3, 4, 5, 6].
+#[allow(clippy::multiple_bound_locations)]
```

Comment:
Instead of adding a Clippy exception, I'd put the `?Sized` to the `W: Write` as `W: Write + Sized`.